crates/core/src/delta_datafusion/expr.rs

```diff
@@ -27,7 +27,7 @@ use std::{
 };
 
 use arrow_schema::DataType;
-use chrono::{Date, NaiveDate, NaiveDateTime, TimeZone};
+use chrono::{DateTime, NaiveDate};
```

Comment:
Why the switch to `DateTime` from `NaiveDateTime`?

Reply:
Cool, thanks for claryfying @ion-elgreco. Would be great to capture these kinds of changes in the commit message when making them.

Reply:
@cmackenzie1 I'll try to make more meaningful commit messages : P